replace.go

```diff
@@ -0,0 +1,9 @@
+// +build !windows
+
+package cacheddownloader
+
+import "os"
+
+func Replace(src, dst string) error {

```

Comment:
Let's not export the `Replace` function from the package unless we actually need it elsewhere.